src/components/MyAvatarMenu/MyAvatarMenu.tsx

```diff
@@ -51,67 +40,78 @@ export const MyAvatarMenu = () => {
   const myProfile = useMyProfile();
   const { icon, address, logout } = useWalletStatus();
 
-  const [anchorEl, setAnchorEl] = React.useState<HTMLElement | null>(null);
-
   return (
     <>
-      <ApeAvatar
-        profile={myProfile}
-        onClick={event => setAnchorEl(event.currentTarget)}
-        className={
-          !anchorEl
-            ? classes.avatarButton
-            : clsx(classes.avatarButton, 'selected')
-        }
-      />
       <Hidden smDown>
-        <Popover
-          anchorEl={anchorEl}
-          anchorOrigin={{ vertical: 'bottom', horizontal: 'right' }}
-          classes={{ paper: classes.popover }}
-          id="my-avatar-popover"
-          onClick={() => setTimeout(() => setAnchorEl(null))}
-          onClose={() => setAnchorEl(null)}
-          open={!!anchorEl}
-          transformOrigin={{ vertical: 'top', horizontal: 'right' }}
-        >
-          <Box
-            css={{
-              display: 'flex',
-              flexDirection: 'column',
-              pt: '$md',
-              '> *': { padding: '$xs $lg' },
-              '> a': {
-                color: '$text',
-                '&:hover': { color: '$black' },
-              },
-            }}
-          >
-            <Box css={{ display: 'flex', alignItems: 'center' }}>
-              <Box css={{ mr: '$sm', display: 'flex' }}>{icon}</Box>
-              {address && shortenAddress(address)}
-            </Box>
-
-            <AppLink to={paths.profile('me')}>My Profile</AppLink>
-            <AppLink to={paths.circles}>My Circles</AppLink>
-            <Link href={EXTERNAL_URL_DOCS}>Docs</Link>
-            <Link css={{ cursor: 'pointer' }} onClick={logout}>
-              Log Out
+        <Popover>
+          <PopoverTrigger asChild>
+            <Link href="#">
+              <ApeAvatar profile={myProfile} className={classes.avatarButton} />
             </Link>
-            <Link
+          </PopoverTrigger>
+          <PopoverContent
+            sideOffset={-66}
+            alignOffset={-16}
```

Comment:
Magic numbers not ideal. Should at least extract into constants and/or add comments explaining how you got to these values, if there's no way to avoid them

Reply:
This is a tough one, as there are no easy calculations here, and we can only use integers (no css values).  IMO, this needs to be dialed in browser.  The problem is that the popover tool on open appends a new dom node to body with the popover positioned with Radix translate3d 'magic'.  Ideally the popover would be positioned absolutely relative to a parent of the trigger since the design shows the popover perfectly overlaying the trigger, but that's just not how this tool works :(  I guess we could roll our own, but that's not insubstantial as a ui task.  I added a short comments in the code.